java/src/main/java/org/upgradeplatform/utils/PublishingRetryCallback.java

```diff
@@ -1,25 +1,36 @@
 package org.upgradeplatform.utils;
+import static org.upgradeplatform.utils.Utils.*;
 
 import javax.ws.rs.client.AsyncInvoker;
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.client.InvocationCallback;
 import javax.ws.rs.core.Response;
 
-
 import static javax.ws.rs.core.Response.Status.Family.SUCCESSFUL;
 
 public class PublishingRetryCallback<T> implements InvocationCallback<Response> {
 
 	private final AsyncInvoker invoker;
-	private final Entity<T> message;
+	private Entity<T> message;
```

Comment:
you can leave this `final` -- you just need to move the `this.message = null;` line into the other constructor :)